tests/integrations/featureflags/test_featureflags.py

```diff
@@ -0,0 +1,142 @@
+import asyncio
+import concurrent.futures as cf
+
+import pytest
+
+import sentry_sdk
+from sentry_sdk.integrations import _processed_integrations, _installed_integrations
+from sentry_sdk.integrations.featureflags import (
+    FeatureFlagsIntegration,
+    add_feature_flag,
+)
+
+
+@pytest.fixture
+def uninstall_integration():
```

Comment:
I found using the `reset_integrations` fixture in `conftest.py` caused issues with the dedupe integration